client/src/views/CreateOrganizationModal.vue

```diff
@@ -193,6 +194,43 @@ const spinnerPage = ref();
 
 const device: Ref<AvailableDevice | null> = ref(null);
 
+interface Title {
+  title: string,
+  subtitle?: string,
+}
+
+const titles = new Map<CreateOrganizationStep, Title>([
+  [
+    CreateOrganizationStep.OrgNameStep,
+    {
+      title: t('CreateOrganization.title.create'),
+      subtitle: t('CreateOrganization.subtitles.nameYourOrg')},
```

Comment:
Closing `}` should be on the next line (same mistake for all following objects).

Or the whole thing could probably be condensed into something like
```
const titles = new Map<CreateOrganizationStep, Title>([[
  CreateOrganizationStep.OrgNameStep, {
    title: t('CreateOrganization.title.create'),
    subtitle: t('CreateOrganization.subtitles.nameYourOrg'),
  }], [
  CreateOrganizationStep.UserInfoStep, {
    title: t('CreateOrganization.title.personalDetails'),
    subtitle: t('CreateOrganization.subtitles.personalDetails'),
  }],
]
```
if the linter is fine with that.

Reply:
I've condensed it, I don't have any linter errors.